CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# How to contribute to this project ?
+
+1. First you must choose a issue on [the project github](https://github.com/mdl29). If you want to do something but no issue were create yet you must create one (try to tag it properly).
```

Comment:
- `a issue` -> `an issue`
- wrong link to the repository : `https://github.com/mdl29` -> `https://github.com/mdl29/scratchy`
- `no issue were create yet` --> `no issue is created yet`